src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/VirtualMachinesOverviewTabDetails.tsx

```diff
@@ -1,7 +1,124 @@
 import * as React from 'react';
 
-const VirtualMachinesOverviewTabDetails = () => {
-  return <div>VirtualMachinesOverviewTabDetails</div>;
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import MutedTextSpan from '@kubevirt-utils/components/MutedTextSpan/MutedTextSpan';
+import { timestampFor } from '@kubevirt-utils/components/Timestamp/utils/datetime';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useVMIAndPodsForVM } from '@kubevirt-utils/resources/vm';
+import {
+  getOperatingSystem,
+  getOperatingSystemName,
+} from '@kubevirt-utils/resources/vm/utils/operation-system/operationSystem';
+import { useGuestOS } from '@kubevirt-utils/resources/vmi';
+import {
+  Card,
+  CardBody,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Divider,
+  Grid,
+  GridItem,
+  Popover,
+  PopoverPosition,
+} from '@patternfly/react-core';
+
+import { getVMStatusIcon } from '../../../../../utils';
+import Flavor from '../../../details/components/Flavor/Flavor';
+
+import VirtualMachinesOverviewTabDetailsConsole from './components/VirtualMachinesOverviewTabDetailsConsole';
+import VirtualMachinesOverviewTabDetailsTitle from './components/VirtualMachinesOverviewTabDetailsTitle';
+
+import './virtual-machines-overview-tab-details.scss';
+
+type VirtualMachinesOverviewTabDetailsProps = {
+  vm: V1VirtualMachine;
+};
+
+const VirtualMachinesOverviewTabDetails: React.FC<VirtualMachinesOverviewTabDetailsProps> = ({
+  vm,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const { vmi } = useVMIAndPodsForVM(vm?.metadata?.name, vm?.metadata?.namespace);
+  const [guestAgentData] = useGuestOS(vmi);
+  const Icon = getVMStatusIcon(vm?.status?.printableStatus);
+
+  const timestamp = timestampFor(
+    new Date(vm?.metadata?.creationTimestamp),
+    new Date(Date.now()),
+    true,
+  );
+
+  return (
+    <div className="VirtualMachinesOverviewTabDetails--details">
+      <Card>
+        <VirtualMachinesOverviewTabDetailsTitle vm={vm} />
+        <Divider />
+        <CardBody isFilled>
+          <Grid>
+            <GridItem span={6}>
+              <DescriptionList isHorizontal>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>{t('Name')}</DescriptionListTerm>
+                  <DescriptionListDescription> {vm?.metadata?.name}</DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                  <DescriptionListTerm>{t('Status')}</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    <Popover
+                      headerContent={<div>{vm?.status?.printableStatus}</div>}
+                      bodyContent={
+                        <div>
+                          {t('Virtual Machines is currently ')} {vm?.status?.printableStatus}
```

Comment:
if I'm not wrong, it should be `Virtual Machine...`

Reply:
cool